src/parallel/worktree-manager.ts

```
@@ -0,0 +1,411 @@
+/**
+ * Sanitize a task ID into a valid git branch name.
+ * Removes/replaces invalid characters and ensures the result is safe for git.
+ */
+function sanitizeBranchName(taskId: string): string {
+  let sanitized = taskId;
+
+  // Replace spaces and invalid characters with dashes
+  sanitized = sanitized.replace(/[\s~^:?*\[\\@{]/g, '-');
+
+  // Remove control characters
+  sanitized = sanitized.replace(/[\x00-\x1f\x7f]/g, '');
```

Biome (2.3.13)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)